src/pawns.cpp

```diff
@@ -195,6 +203,10 @@ namespace {
     b = e->semiopenFiles[Us] ^ 0xFF;
     e->pawnSpan[Us] = b ? int(msb(b) - lsb(b)) : 0;
 
+    // Center binds: Two pawns controlling the same central square
+    const Bitboard pawnBinds = shift_bb<Right>(ourPawns) & shift_bb<Left>(ourPawns);
```

Comment:
I would reuse 'b' instead of intorducing

const Bitboard pawnBinds

This is how it is done in teh above code.